docs/api/browser-window.md

```diff
@@ -6,21 +6,26 @@ Process: [Main](../glossary.md#main-process)
 
 ```javascript
 // In the main process.
-const { BrowserWindow } = require('electron')
+const { app, BrowserWindow } = require('electron')
 
 // Or use `remote` from the renderer process.
 // const { BrowserWindow } = require('electron').remote
 
-let win = new BrowserWindow({ width: 800, height: 600 })
-win.on('closed', () => {
-  win = null
-})
+let win
 
-// Load a remote URL
-win.loadURL('https://github.com')
+app.on('ready', () => {
+  win = new BrowserWindow({ width: 800, height: 600 })
 
-// Or load a local HTML file
-win.loadURL(`file://${__dirname}/app/index.html`)
+  // Load a remote URL
+  win.loadURL('https://github.com')
+
+  // Or load a local HTML file
+  win.loadURL(`file://${__dirname}/app/index.html`)
+
+  win.on('closed', () => {
+  win = null
```

Comment:
This line needs to be properly indented in order to pass our linting tests.  You can verify the change locally by making the change and running `npm run lint:docs`
```suggestion
    win = null
```

Reply:
Done. Thanks!